crossbuilder

```diff
@@ -196,8 +200,10 @@ setup_lxd () {
         echo
         if [ "$REPLY" = y ]
         then
-            echo "${POSITIVE_COLOR}Installing ZFS.${NC}"
-            sudo apt-get install -y zfsutils-linux
+            if ! which ztool > /dev/null ; then
```

Comment:
Isn't it `zpool`? https://linux.die.net/man/8/zpool

Reply:
Indeed! Should be fixed, now.